agent/config/config.go

```diff
@@ -239,7 +239,8 @@ func NewConfig(ec2client ec2.EC2MetadataClient) (*Config, error) {
 		ec2client = ec2.NewBlackholeEC2MetadataClient()
 	}
 
-	config.determineIPCompatibility(ec2client)
+	// TODO - Enable when launching IPv6-only support
+	// config.determineIPCompatibility(ec2client)
```

Comment:
Can we use a searchable tag like: `feat:IPv6-only` and include it in all todos marked here to make unwinding/reverting easier?

Reply:
sure

Reply:
Updated. Thanks!